### DISCUSSION TITLE
UserTask can specify user processing

### DISCUSSION THREAD

**Discussion by Nokecy** [elsa-workflows/elsa-core#1328] [Q&A] [answered]

**Comment by sfmskywalker** [comment]
Hi @Nokecy, thanks for opening an issue. But, what is it about? 🤔

**Comment by Nokecy** [comment]
https://github.com/elsa-workflows/elsa-core/issues/241
We have a document approval workflow with e.g. document1 to be approved by userA, document2 to be approved by userB and document3 to be approved by userA
How to set userA、userB、userC

**Comment by Nokecy** [comment, marked as answer]
https://github.com/elsa-workflows/elsa-core/discussions/475

**Reply by sfmskywalker** [reply]
The answer in #475 seems appropriate to your question as well. Let me know if you have any follow-up questions.
